### PR TITLE
Accept "newest" as input for --kubernetes-version

### PR DESCRIPTION
**Pull request by sgmac**
Adds support for #13076

# Testing

```shell
 ./out/minikube start -p testin1 --kubernetes-version=newest
* [testin1] minikube v1.24.0 on Ubuntu 20.04
* Automatically selected the docker driver. Other choices: kvm2, virtualbox, none, ssh
* Starting control plane node testin1 in cluster testin1
* Pulling base image ...
* Downloading Kubernetes v1.23.0-beta.0 preload ...
    > preloaded-images-k8s-v14-v1...: 506.33 MiB / 506.33 MiB  100.00% 25.33 Mi
    > gcr.io/k8s-minikube/kicbase: 355.78 MiB / 355.78 MiB  100.00% 9.17 MiB p/
* Creating docker container (CPUs=2, Memory=16000MB) ...
* Preparing Kubernetes v1.23.0-beta.0 on Docker 20.10.8 ...
  - Generating certificates and keys ...
  - Booting up control plane ...
  - Configuring RBAC rules ...
* Verifying Kubernetes components...
  - Using image gcr.io/k8s-minikube/storage-provisioner:v5
* Enabled addons: default-storageclass, storage-provisioner
* Done! kubectl is now configured to use "testin1" cluster and "default" namespace by default
```

### CODE REVIEW

File: cmd/minikube/cmd/start.go [hunk 1]
```diff
@@ -1577,7 +1577,7 @@ $ minikube config unset kubernetes-version`)
 
 	if paramVersion == "" || strings.EqualFold(paramVersion, "stable") {
 		paramVersion = constants.DefaultKubernetesVersion
-	} else if strings.EqualFold(paramVersion, "latest") {
+	} else if strings.EqualFold(paramVersion, "latest") || strings.EqualFold(paramVersion, "newest") {
```
Comment (medyagh) [inline cmd/minikube/cmd/start.go:1580]: while we are at it, how about we accept both lower case and upper case

strings.EqualFold(strings.ToLower(paramVersion), "latest") ...(and same for other one)



strings.EqualFold(paramVersion, "latest")

**Review by medyagh — CHANGES_REQUESTED** [review]

File: cmd/minikube/cmd/start.go (hunk 1 above)
Comment (sgmac) [reply cmd/minikube/cmd/start.go:1580]: @medyagh good idea, I'll PR changes.

Comment (spowelljr) [reply cmd/minikube/cmd/start.go]: @medyagh `strings.EqualFold` is case-insensitive already

**Comment by medyagh** [comment]
Thank you for this pr

**Pull request merged**